api/v1/verticadb_webhook.go

```diff
@@ -116,7 +119,11 @@ func (v *VerticaDB) ValidateUpdate(old runtime.Object) error {
 	verticadblog.Info("validate update", "name", v.Name, "GroupVersion", GroupVersion)
 
 	allErrs := append(v.validateImmutableFields(old), v.validateVerticaDBSpec()...)
-	if allErrs == nil {
+	allErrs = v.validateShutdownSandboxImage(old, allErrs)
```

Comment:
The new functions should be put in validateImmutableFields().

Reply:
moved. Three unrelated test cases were broken and fixed to pass the newly added tests.